### PR TITLE
Update footer form

### PR DESCRIPTION
**Pull request by mitchellwright**
Update the footer email subscription form to submit to a Tray.io endpoint instead of ConvertKit

### CODE REVIEW

File: docs/pages/api/signup.tsx [hunk 1]
```diff
@@ -1,23 +1,31 @@
 import { NextApiRequest, NextApiResponse } from "next";
-import { subscribeToForm } from "../../lib/ConvertKitApi";
 
-const FORM_ID = process.env.CONVERTKIT_FORM_ID;
+const TRAY_URL = "https://39dca6c2-9ca4-41b4-82c9-e48202f221f8.trayapp.io";
```
Comment (mitchellwright) [inline docs/pages/api/signup.tsx:3]: I can move this into an env var if that's preferred

Comment (jaredpalmer) [reply docs/pages/api/signup.tsx:3]: Yes please. Can you generate a new one and add it as env var since this URL is compromised now?

**Comment by mitchellwright** [comment]
Ready for review again @jaredpalmer

**Comment by jaredpalmer** [comment]
Looks good, can you share w/me endpoints in Slack DM or 1Password so that I can similarly update the discord bot?

**Review by jaredpalmer — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**